File: server/src/main/java/ch/zxseitz/tbsg/server/model/Role.java
package ch.zxseitz.tbsg.server.model;

import org.springframework.security.core.GrantedAuthority;
import org.springframework.security.core.authority.SimpleGrantedAuthority;

public class Role {
    public static final GrantedAuthority Admin = new SimpleGrantedAuthority("Admin");
    public static final GrantedAuthority User = new SimpleGrantedAuthority("User");
}

File: server/src/main/java/ch/zxseitz/tbsg/server/model/Role.kt
package ch.zxseitz.tbsg.server.model;

import org.springframework.security.core.GrantedAuthority;
import org.springframework.security.core.authority.SimpleGrantedAuthority;

enum class Role(val authority: GrantedAuthority) {
    Admin(SimpleGrantedAuthority("Admin")),
    User(SimpleGrantedAuthority("User"))
}
